OpenRA.Game/Traits/Player/Shroud.cs

```diff
@@ -17,22 +17,52 @@ namespace OpenRA.Traits
 	[Desc("Required for shroud and fog visibility checks. Add this to the player actor.")]
 	public class ShroudInfo : ITraitInfo, ILobbyOptions
 	{
+		[Translate]
+		[Desc("Label for the fog checkbox in the lobby.")]
+		public readonly string FogLabel = "Fog of War";
+
+		[Translate]
+		[Desc("Description of the fog checkbox in the lobby.")]
+		public readonly string FogDescription = "Line of sight is required to view enemy forces";
+
 		[Desc("Default value of the fog checkbox in the lobby.")]
-		public bool FogEnabled = true;
+		public readonly bool FogEnabled = true;
 
 		[Desc("Prevent the fog enabled state from being changed in the lobby.")]
-		public bool FogLocked = false;
+		public readonly bool FogLocked = false;
+
+		[Desc("Display the fog checkbox in the lobby.")]
+		public readonly bool FogVisible = true;
```

Comment:
This is going to be so confusing...

Reply:
Have to agree here, can't we call those `FogCheckboxVisible`, `ShroudCheckboxVisible` and so on?

Reply:
Not without renaming all the other fields related to these across all the traits that use them.  I'm not willing to do that after three weeks of this sitting idle, so if this is a blocker then close the PR.

Reply:
If you close this then please also revert the superstupid "hey-we're-overlooking-how-hacky-these-are-cuz-guy-is-newcomer" checkbox PRs.

Reply:
> If this is a blocker for this PR, then please push an amendment over this if you want to address it yourself, or we can close it.

It would have been nicer, but it's not a blocker to me, and Fog + Shroud are the only ambiguous cases anyway. My +1 still stands.